easybuild/easyconfigs/h/HIPS/HIPS-1.2b-rc5-foss-2017b.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'MakeCp'
+
+name = 'HIPS'
+version = '1.2b-rc5'
+
+homepage = 'http://hips.gforge.inria.fr/'
+description = """ HIPS (Hierarchical Iterative Parallel Solver) is a scientific library that provides
+ an efficient parallel iterative solver for very large sparse linear systems. """
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['http://hips.gforge.inria.fr/release/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['db198f65dd4bbca25cffe537e1c659c7ca94cd5a64a32c8b9165caacb5b09603']
+
+dependencies = [
+    ('SCOTCH', '5.1.12b_esmumps'),
+]
+
+# options in makefile.inc are adapted from Makefile_Inc_Examples/makefile.inc.gnu
+prebuildopts = 'echo "COEFTYPE = -DTYPE_REAL" >makefile.inc && '
+prebuildopts += 'echo "PARTITIONER = -DSCOTCH_PART" >>makefile.inc && '
+prebuildopts += 'echo "ARCH = -DLINUX" >>makefile.inc && '
+prebuildopts += 'echo "LD = \$(FC)" >>makefile.inc && '
+prebuildopts += 'echo "MPILD = \$(MPIFC)" >>makefile.inc && '
+prebuildopts += 'echo "LBLAS = $LIBBLAS" >>makefile.inc && '
+prebuildopts += 'echo "MAKE = make" >>makefile.inc && '
+prebuildopts += 'echo "AR = ar" >>makefile.inc && '
+prebuildopts += 'echo "ARFLAGS = -crs" >>makefile.inc && '
+prebuildopts += 'echo "LN = ln" >>makefile.inc && '
+prebuildopts += 'echo "CP = cp" >>makefile.inc && '
```

Comment:
Hmm, isn't it easier to just copy `makefile.inc.gnu` to `makefile.inc` via `prebuildopts`, and then override values via `buildopts` like this:

```python
buildopts = 'ISCOTCH="-I$EBROOTSCOTCH/include" LSCOTCH="-L$EBROOTSCOTCH/lib -lscotch -lscotcherr" '

Reply:
cool, I did not know you could do that